lib/rules/declaration-property-value-no-unknown/__tests__/index.mjs

```diff
@@ -249,13 +255,24 @@ testRule({
 			endLine: 1,
 			endColumn: 13,
 		},
+		{
+			code: stripIndent`a {
+				top: /* a comment */
+					red;
+			}`,
+			message: messages.rejected('top', 'red'),
+			line: 3,
+			column: 6,
+			endLine: 3,
+			endColumn: 9,
+		},
 		{
 			code: 'a { color: ; }',
 			message: messages.rejected('color', ''),
 			line: 1,
-			column: 11,
+			column: 12,
 			endLine: 1,
-			endColumn: 12,
+			endColumn: 13,
```

Comment:
Spaces are valid.
It is the unexpected early `;` that is invalid.

Reply:
l202–209 is a variant without the space